packages/office-ui-fabric-react/src/components/__snapshots__/Facepile.AddFace.Example.tsx.shot

```diff
@@ -2,16 +2,40 @@
 
 exports[`Component Examples renders Facepile.AddFace.Example.tsx correctly 1`] = `
 <div
-  className="ms-Facepile"
+  className=
+      ms-Facepile
+      {
+        width: auto;
+      }
 >
```

Comment:
Not sure why these Facepile snapshots updated.  Looks like they go from undefined to styled so that may be good, but might want to double check before merge

Reply:
I'm betting it was from the Facepile mergestyles update from a few days ago. I don't know why they're updating now, maybe I neglected to run them properly on the actual Facepile MergeStyles PR. What would I want to double check to be sure this is okay for merge?

Reply:
Probably good if it passes VR tests, but if it were me I'd look for the style sets in Facepile to see they match so I could rule out styles from Datepicker funneling into Facepile for some crazy reason.